### ISSUE TITLE
Consecutive note durations cut-off after first note

### ISSUE THREAD

**Issue by Tony-Pappas** [melanchall/drywetmidi#111]
Hi, I'm trying to get started with drywebmidi, so I may be doing something wrong, but I can't seem to figure out why all consecutive notes are getting abruptly cut-off after the first note. It's as if the note-off is getting sent right after the note-on. But I'm not seeing that when tracing out on outputDevice.EventSent.

Is anyone able to reproduce this? Just take a standard type 1 midi file and put four whole notes on G5. Here's my test file: [test-whole-single.zip](https://github.com/melanchall/drywetmidi/files/5976214/test-whole-single.zip)

Here's my code (.NET Standard version):
```C#
outputDevice = OutputDevice.GetByName(outputDeviceName);

var file = @"A:\Code\TestFiles\test-whole-single.mid";
midiFile = MidiFile.Read(file);

var tracks = midiFile.GetTrackChunks();
track = tracks.ElementAt(1);

var division = new TicksPerQuarterNoteTimeDivision(960);
var tempo = Tempo.FromBeatsPerMinute(100);
playback = track.GetPlayback(TempoMap.Create(division, tempo), outputDevice);
playback.Loop = true;
playback.Start();
```

**Comment by melanchall** [comment]
Hi,

Thanks for using the library. Unfortunately I can't see any issue with your code. I've created new solution, placed your code with your file into it and run the program. All notes are played correctly.

Here my code: [kPlaybackIssue.zip](https://github.com/melanchall/drywetmidi/files/5977765/kPlaybackIssue.zip). Please check it out. Do you have any additional code that can affect playback? Can you please record what do you hear and send me that recording?

**Comment by Tony-Pappas** [comment]
Hi Maxim,

Thank you for your quick response!

Okay, I looked into this again and it looks like there's a bug in Cakewalk TTS-1. It's working fine with other samplers. I thought it was something *I* was doing wrong, I should have done more research before posting.

So, I've worked with DryWetMIDI this weekend... and Maxim, this software is superb. I've been coding for many years, used a lot of midi libraries and written a few, but yours is exceptional. The way you've implemented time is beautiful and I love your TimeSpan abstraction. I like the Player, and the PlaybackCurrentTimeWatcher is a great idea. 

Everything has tests on it, your coding style is top, I could go on and on, you've done brilliant work here!

Thank you for sharing this with us! I hope to make some kind of contribution soon.

Thanks again Maxim!

-Tony
